.travis.yml

```diff
@@ -1,22 +1,36 @@
 language: python
-matrix:
-  include:
-    - python: "2.6"
-      env: TOXENV=py26
-    - python: "2.7"
-      env: TOXENV=py27
-    - python: "3.4"
-      env: TOXENV=py34
-    - python: "3.5"
-      env: TOXENV=py35
-# command to install dependencies
-install:
-  - "pip install tox"
-  - "pip install coveralls"
-# command to run tests
-script: tox
-# run in a docker container
-sudo: false
+sudo: "required"
+services:
+  - docker
+env:
+  matrix:
+    - OS=centos
+      OS_VERSION=6
+      PYTHON_VERSION=2
+    - OS=centos
+      OS_VERSION=7
+      PYTHON_VERSION=2
+    - OS=fedora
+      OS_VERSION=25
+      PYTHON_VERSION=2
+    - OS=fedora
+      OS_VERSION=25
+      PYTHON_VERSION=3
+    - OS=fedora
+      OS_VERSION=26
+      PYTHON_VERSION=2
+    - OS=fedora
+      OS_VERSION=26
```

Comment:
Since 27 is released, I think it would make sense to add it to the list.

Reply:
Rawhide is sufficiently close to 27 now, I was thinking it makes sense to wait until F25 goes EOL and replace it with 27.

Otherwise we'll have an increased test length and probably some 27-specific hacks

Reply:
...and we'll be sure the software works on latest platforms. Isn't that the reason to use CI?

+1 to do s/25/27/

anyway, I don't want my comment to block this PR